docs/examples/webpack/webpack.config.js

```diff
@@ -1,51 +1,109 @@
-const path = require('path')
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const { dirname, resolve } = require('path')
 
-const config = {
-  mode: 'production',
-  devtool: 'source-map',
-  optimization: {
-    minimizer: [
-      new UglifyJsPlugin({
-        cache: true,
-        parallel: true,
-        sourceMap: true,
-        uglifyOptions: {
-          compress: {
-            ie8: true,
-            warnings: false
-          },
-          mangle: {
-            ie8: true
-          },
-          output: {
-            comments: false,
-            ie8: true
-          }
-        }
-      })
-    ]
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const CopyPlugin = require('copy-webpack-plugin')
+const TerserPlugin = require('terser-webpack-plugin')
+
+// Module resolution
+const frontendPath = dirname(require.resolve('govuk-frontend'))
+const html5shivPath = dirname(require.resolve('html5shiv'))
+const webpackPath = dirname(require.resolve('webpack/package.json'))
+const modulesPath = resolve(webpackPath, '../')
+
+module.exports = ({ WEBPACK_SERVE }, { mode }) => ({
+  devServer: {
+    watchFiles: {
+      paths: ['**/*.html']
+    }
   },
-  entry: './assets/main.js',
+
+  devtool: WEBPACK_SERVE
+    ? 'inline-source-map'
+    : 'source-map',
+
+  entry: [
+    './assets/javascripts/main.mjs',
+    './assets/stylesheets/app.scss',
+    './assets/stylesheets/app-ie8.scss'
+  ],
+
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.(cjs|js|mjs)$/,
         exclude: /node_modules/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['env', 'es3']
+        use: ['babel-loader']
+      },
+      {
+        generator: {
+          outputPath: 'assets/stylesheets',
+          publicPath: '/assets/stylesheets',
+          filename: '[name].css'
+        },
+        test: /\.scss$/,
+        type: 'asset',
+        use: [
+          'postcss-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              sassOptions: {
+                quietDeps: true
```

Comment:
https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#silence-deprecation-warnings-from-dependencies-in-dart-sass